AAI/AAIConnectProfile.md

```diff
@@ -230,15 +203,16 @@ upstream.
 
         2.  Pragma: no-cache
         
-5.  MUST provide protection againt Client attacks as outlined in [RFC 6819](https://tools.ietf.org/html/rfc6819).
+5.  MUST provide protection againt Client attacks as outlined in
```

Comment:
Nit:Against, not againt

Reply:
Not quite sure why I can't spell this word so consistently, but thanks for catching it.